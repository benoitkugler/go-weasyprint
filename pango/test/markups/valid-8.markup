<span foreground="blue" alpha="65535">Text</span>
<span foreground="blue" fgalpha="50%">Text</span>
<span background="blue" background_alpha="1">Text</span>
<span background="blue" bgalpha="33%">Text</span>
