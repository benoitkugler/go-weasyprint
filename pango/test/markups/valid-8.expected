Text
Text
Text
Text


---

range 0 4
[0,4]foreground=#00000000ffff
[0,4]foreground-alpha=65535
range 4 5
range 5 9
[5,9]foreground=#00000000ffff
[5,9]foreground-alpha=32767
range 9 10
range 10 14
[10,14]background=#00000000ffff
[10,14]background-alpha=1
range 14 15
range 15 19
[15,19]background=#00000000ffff
[15,19]background-alpha=21626
range 19 2147483647


---

[0:4] (null) Normal
[4:5] (null) Normal
[5:9] (null) Normal
[9:10] (null) Normal
[10:14] (null) Normal
[14:15] (null) Normal
[15:19] (null) Normal
[19:2147483647] (null) Normal
